resources/icons/main/keycard-logo.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
but we don't use svg   anymore

Reply:
@flexsurfer how to implement Select key storage screen then? There is gray/blue color icon toggle  happen on click. For "This device" option there is an SVG icon use so i did same for "Keycard" option. 

<img width="311" alt="Screenshot 2019-08-01 19 59 40" src="https://user-images.githubusercontent.com/23836/62312463-e87b4e80-b496-11e9-822a-fb15f7217bcb.png">

Reply:
i mean we use pngs instead

Reply:
we keep svg only for desktop, but i believe we'll remove them soon

Reply:
ok, i'll change it to png